addons/survey/wizard/survey_invite.py

```diff
@@ -151,10 +161,16 @@ def create(self, vals_list):
     @api.depends('template_id', 'partner_ids')
     def _compute_subject(self):
         for invite in self:
-            langs = set(invite.partner_ids.mapped('lang')) - {False}
-            if len(langs) == 1:
-                invite = invite.with_context(lang=langs.pop())
-            super(SurveyInvite, invite)._compute_subject()
+            if invite.subject:
+                return
```

Comment:
Continue instead of return ? Otherwise you break the loop instead of going into next item of record set :)